api/views/auth/social_auth_complete.py

```
@@ -0,0 +1,252 @@
+                    "email": user.email,
+                    "first_name": user.first_name,
+                    "last_name": user.last_name,
+                    "is_staff": user.is_staff,
+                    "is_superuser": user.is_superuser,
+                },
+                "profile": profile_data,
+            }
+
+            return Response(custom_data, status=status.HTTP_200_OK)
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)